modules/core/src/main/scala/com.snowplowanalytics.snowplow.lakes/tables/DeltaWriter.scala

```diff
@@ -33,6 +35,10 @@ class DeltaWriter(config: Config.Delta) extends Writer {
     )
 
   override def prepareTable[F[_]: Sync](spark: SparkSession): F[Unit] = {
+    val checkHost = Sync[F].blocking {
+      InetAddress.getByName(config.location.getHost())
```

Comment:
The stack trace for a wrong storage name contains
```
[error] Caused by: java.net.UnknownHostException: lbwwlakeloaderaccountfoo.dfs.core.windows.net
[error] 	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:229)
[error] 	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
[error] 	at java.base/java.net.Socket.connect(Socket.java:609)
[error] 	at org.wildfly.openssl.OpenSSLSocket.connect(OpenSSLSocket.java:563)
[error] 	at java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:177)
[error] 	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:509)
[error] 	at java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:604)
[error] 	at java.base/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:266)
[error] 	at java.base/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:373)
```
so which is also in `java.net`